packages/lys/src/slice.ts

```diff
@@ -117,5 +141,25 @@ export const instantiateSlice = <S extends Slice<any, any>>(
     });
   };
 
-  return { state, actions: proxyActions };
+  const proxySelectors: any = {};
+  let latestState: any = null;
+  const selectorCache = new Map();
```

Comment:
I wonder if the use case for Lys could be that heavy of a recomputation process.